cli/parser/parser.go

```diff
@@ -321,6 +316,10 @@ type OptionDefinition struct {
 	// --subcommands false" is actually equivalent to "bazel build
 	// --subcommands=true //false:false".
 	RequiresValue bool
+
+	// PluginID is the ID of the plugin associated with this option definition, if
+	// applicable.
+	PluginID string
```

Comment:
maybe worth clarifying what "plugin" means here - is it a bazel concept? (we also have our own concept of plugins, so this may be ambiguous)

Reply:
Ah, no, it's our concept. I'll clarify it.